params/config.go

```diff
@@ -581,6 +582,13 @@ func (s ScrollConfig) String() string {
 		s.UseZktrie, maxTxPerBlock, maxTxPayloadBytesPerBlock, s.FeeVaultAddress, s.L1Config.String())
 }
 
+func (s ScrollConfig) GetMaxTxPerBlock() int {
+	if s.MaxTxPerBlock == nil {
+		return math.MaxInt
+	}
+	return *s.MaxTxPerBlock
```

Comment:
How would you set this? The tricky part is that we might discard a whole account after certain errors (ccc, nonce, etc). That's why I was considering limiting the results in `Pending` based on the number of accounts, not the number of transactions, but neither are a perfect solution.

Reply:
> The tricky part is that we might discard a whole account after certain errors

I agree. But I considered such scenarios to be rare and didn't want to complicate the logic added here. Maybe we can simply multiply MaxTxPerBlock with some N, so we have higher chance of filling the block even if we end up skipping some accounts.

Reply:
> limiting the results in Pending based on the number of accounts

this will probably yield a more fluctuating cost between each `Pending` call, but might make more sense from UX standpoint. Otherwise, a user with a huge backlog of txns might fill an entire block.

Reply:
> limiting the results in Pending based on the number of accounts

done